.github/workflows/training-model-mgmt-unittests.yaml

```diff
@@ -0,0 +1,53 @@
+name: training-model-mgmt-unittests
+
+on:
+  # push:
+  #   branches:
+  #     - release
+  pull_request:
+    branches:
+      - main
+      - ayushmishra/update_mlflow_convertor_params
+    paths:
+      - assets/training/model_management/*
+      - .github/workflows/training-model-mgmt-unittests.yaml
+  workflow_dispatch:
+    inputs:
+      asset_dirs:
+        description: Asset directories
+        default: assets
+        required: true
+
+env:
+  model_mgmt_root_dir: assets/training/model_management
+  model_mgmt_src_dir: assets/training/model_management/src/azureml/
+  model_mgmt_test_dir: assets/training/model_management/tests
+  model_mgmt_unittests_dir: assets/training/model_management/tests/unittests
+  model_mgmt_test_req: assets/training/model_management/tests/dev_requirements.txt
+
+jobs:
+  run_unit_tests:
+    name: Run
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone branch
+        uses: actions/checkout@v3
+
+      - name: Use Python 3.8
+        uses: actions/setup-python@v4
+        with:
+          python-version: '==3.8'
+
+      - name: Install dev requirements
+        run: |
+          apt-get update && apt-get upgrade && apt-get install -y build-essential
+          pip install -r $model_mgmt_test_req
+
+      - name: Copy dir
+        run: |
+          cp -r $model_mgmt_src_dir $model_mgmt_unittests_dir
+          ls -l $model_mgmt_unittests_dir
+
+      - name: Execute tests
+        run: python -m pytest $model_mgmt_unittests_dir --tb=native --junitxml=./TEST-TEST.xml -x -n 1 -ra --show-capture=no
```

Comment:
Do you want to publish test results to the PR? See [this](https://github.com/Azure/azureml-assets/blob/e4ddb9853ab1995cf0ea97df2c094322174b6190/.github/workflows/scripts-test.yaml#L79-L84) for an example.

Reply:
Thanks @elesel. Added

Reply:
workflow is failing with: 

![image](https://github.com/Azure/azureml-assets/assets/61145377/9263aec7-e073-44cf-9d16-8dca34f02ef8)

Are you aware why this error is coming?